atsign-foundation/atsign.dev-3.0
# Added more content to encryption, public and private keys as well as …

…fixed privacy lead too long issue. Also changed the short-code to fix issue where image looked stretched. Lastly, fixed issue where single page image was in asset folder.



**- What I did**
- Added more content that related with how AtSign implemented encryption and keys.
- Removed mistake where the lead for the privacy page was too long with a shorter sentence.
- I changed the short code for the *image.html* document to use img-contain which prevents an issue with unbounded images.
**- How I did it**

See screenshots below:
![image](https://user-images.githubusercontent.com/34658202/172952413-2e9ec719-7989-4b94-8711-3ade2c647719.png)
![image](https://user-images.githubusercontent.com/34658202/172952489-44894424-2759-46e1-b25b-f702dc557f46.png)
![image](https://user-images.githubusercontent.com/34658202/172952552-553d82ee-1915-43da-9ace-197f58e2bc2b.png)

- I removed the directory from assets and made a new folder for the content in public_private_keys where the image should be.
- Lastly, I stopped using the showcase card shortcode and applied img-contain to the image shortcode which fixed the issue.

**- How to verify it**
- See screenshots to double check that content is up to date with website. 
- Check directory folder in assets to make sure that E2EE.png is gone.
- The image in public_private_keys should look normal now and not stretched.

**- Description for the changelog**
